### PR TITLE
html indezx created

### PR DESCRIPTION
**Pull request by bilalelbedoui93**

### CODE REVIEW

File: staff/bilal-elbedoui/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV</title>
+</head>
+<body>
+    <div class="head">
```
Comment (lexbarba) [inline staff/bilal-elbedoui/cv/index.html]: @bilalelbedoui93 div is not a semantic tag. please remove it.

File: staff/bilal-elbedoui/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        <header>
+            <h1>CURRICULUM VITAE</h1>         
+        </header>
+        <nav>
+            <li><a href="#">About</a></li>
```
Comment (lexbarba) [inline staff/bilal-elbedoui/cv/index.html]: @bilalelbedoui93  all li elements should be inside an ol or ul tag. please update accordingly.

File: staff/bilal-elbedoui/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            </figure>
+
+            <figure>
+                <img>
+                <figcaption>linkedin</figcaption>
+            </figure>    
+
+        </section>
+
+            <div class="line"></div>
```
Comment (lexbarba) [inline staff/bilal-elbedoui/cv/index.html]: @bilalelbedoui93 no divs please.

**Review by lexbarba — CHANGES_REQUESTED** [review]

File: staff/bilal-elbedoui/cv/index.html (hunk 3 above)
Comment (bilalelbedoui93) [reply staff/bilal-elbedoui/cv/index.html]: removed too!!!

File: staff/bilal-elbedoui/cv/index.html (hunk 2 above)
Comment (bilalelbedoui93) [reply staff/bilal-elbedoui/cv/index.html]: I added it!!!

File: staff/bilal-elbedoui/cv/index.html (hunk 1 above)
Comment (bilalelbedoui93) [reply staff/bilal-elbedoui/cv/index.html]: removed!!!

**Pull request merged**